EjemploFiguras/Circle.java

```diff
@@ -0,0 +1,39 @@
+package EjemploFiguras;
+
+class Circulo extends Shape {
+    double radius;
+
+    @Override double area(){
+        this.area = 3.1416*this.radius;
+        return this.area;
+    }
+
+    @Override public String getColor() {
+        return "Color: " + color;
+    }
+
+    public String setColor(String colorAsignado){
+        this.color = colorAsignado;
+        return this.color;
```

Comment:
The setters do not have a return, change it to void and remove the return.

Reply:
Ohhhh true, let me fix that :)